components/Footer.js

```diff
@@ -206,22 +206,24 @@ export default function Footer() {
           </div>
         </div>
         <div className="mt-8 border-t border-white/10 pt-8 flex flex-col md:flex-row   items-center justify-between">
-          <div className="flex pb-4 md:pb-0 items-center justify-center space-x-6 md:order-2">
-            <p className=" text-xs leading-5 text-primary-low-high  md:mt-0">
+          <div className="flex pb-4 md:pb-0 items-center justify-center space-x-6 md:order-2 flex-wrap">
```

Comment:
Hello! Thank you for working on this.  It is looking nice, but the icons are not centered on mobile under the text.  This appears to be due to the space-x-6 on this div creating a space that is not needed on mobile, but remains needed on desktop in the current configuration..  There are a few ways you can approach addressing this, but take a look and see if you can ensure the icons are centered on mobile.

![left space](https://github.com/EddieHubCommunity/BioDrop/assets/97615019/f69541c1-c1bb-437a-81db-802de7b5e50d)

Reply:
Hey @amandamartin-dev  the changes are made as you said. Please review it incase of any query please tell me. Thank you.

Reply:
<img width="348" alt="Screenshot 2023-10-10 100338" src="https://github.com/EddieHubCommunity/BioDrop/assets/143615010/c956a176-d114-4406-8d1e-5c7fef3c8225">
<img width="284" alt="Screenshot 2023-10-10 100356" src="https://github.com/EddieHubCommunity/BioDrop/assets/143615010/ccbd1f48-14e3-450c-a640-ed1b8d11dc98">